index.html

```diff
@@ -65,6 +74,7 @@
 		<img class="desc-image" src="{{ "images/laser.png" | relative }}" />
 		<div class="desc-text">
 			<p>
+				<!-- I would list more than two modules here; probably change the picture to something displaying more than just the image -->
```

Comment:
Seeing as this section is now about more things than lasers, do people have suggestions for a better name?